api/genomics/src/main/java/org/pmiops/workbench/genomics/RandomizeVcf.java

```diff
@@ -142,6 +141,8 @@ protected Genotype randomizeGenotype(VariantContext variantContext, Genotype gen
     double genotypeTypeIndex = random.nextDouble();
     if (genotypeTypeIndex < .0145) {
       // double no-call
+      alleles.add(Allele.NO_CALL);
```

Comment:
are these `NO_CALL`s fixing a different issue?

Reply:
Yes - `VcfWriter` gets unhappy when some samples have no alleles when there are samples that do have alleles.